Artifacts/MsSql/Structure/Security/0040-Tables - RLPhase2.sql

```diff
@@ -0,0 +1,132 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+---------------------------- [dbo].[ClaimSetResourceClaimActionAuthorizations] --------------------------------------------------------
+
+CREATE TABLE [dbo].[ClaimSetResourceClaimActionAuthorizations](
```

Comment:
One thing that has been missing in the current security metadata model is appropriate unique constraints.

For `dbo.ClaimSetResourceClaimActionAuthorizations`, we should have a unique constraint on `ClaimSet_ClaimSetId`, `ResourceClaim_ResourceClaimId` and `Action_ActionId`.

@vimayya Thoughts?

Reply:
Added the unique constraint in the scripts file, do I need to add the change in security context in the security data access project